lib/src/components/NxTreeView/NxTreeView.tsx

```diff
@@ -47,7 +47,7 @@ const NxTreeView: FunctionComponent<Props> =
         triggerTooltipProps = typeof triggerTooltip === 'string' ? { title: triggerTooltip } : triggerTooltip;
 
     return (
-      <div className={treeViewClasses} id={treeViewId} role="tree">
+      <div className={treeViewClasses} id={treeViewId} role="list">
```

Comment:
> I'd like someone with VoiceOver to test this. It doesn't do great in ChromeVox since it always reads it as a "list of zero items". I don't think CV understands the `list -> group -> listitem` structure even though that is standards-compliant

Tested this with VoiceOver and can confirm that it works well and says the correct number of list items.